src/api/Jalasoft.TeamUp.Resumes.API/local.settings.json

```diff
@@ -6,5 +6,9 @@
   },
   "Host": {
     "CORS": "*"
+  },
+
+  "appSettings": {
+    "connectionString": "Server = DESKTOP-L33PS1L; DataBase = TeamUp; User ID = AppConnection; Password=12345678; "
```

Comment:
Instead of creating appSettings please insert the SQL connections string in the Values, just like we did in Projects

Reply:
Insert it.